airbyte/plural/docs/running-on-custom-nodes.md

```diff
@@ -0,0 +1,61 @@
+## Running Airbyte Workloads on Custom Node Group
+
+There may be a desire to run your Airbyte workloads on a specific node size for large workloads, or maybe even 
+[spot instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html) to achieve higher cost 
+savings.
+
+### create custom node group
```

Comment:
We can probably leave this part out of the airbyte docs (or explain its optional as well only if you want to ensure a 3 node spot pool)